tests/providers/microsoft/azure/hooks/test_adls_v2.py

```diff
@@ -0,0 +1,108 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
If the hook logic is in `hooks/data_lake.py` why do we create a `test_adls_v2.py` ?

Reply:
Yeah @eladkal I am moving the tests to `test_azure_data_lake` file. working on it

Reply:
Moved the test to `test_azure_data_lake` file

Reply:
@eladkal Can you review it now